fwup-ops.conf

```diff
@@ -36,28 +43,37 @@ define(NERVES_FW_AUTHOR, "The Nerves Team")
 # +----------------------------+
 # | MBR                        |
 # +----------------------------+
-# | boot0                      |
-# +----------------------------+
-# | boot0 copy (unused)        |
+# | SPL and U-Boot             |
 # +----------------------------+
 # | U-Boot environment         |
 # +----------------------------+
-# | U-Boot                     |
-# +----------------------------+
-# | U-Boot copy (unused)       |
-# +----------------------------+
 # | p1: Rootfs A (squashfs)    |
 # +----------------------------+
 # | p2: Rootfs B (squashfs)    |
 # +----------------------------+
 # | p3: Application (f2fs)     |
 # +----------------------------+
 
+define(UBOOT_OFFSET, 16)
+define(UBOOT_COUNT, 8176)
+
 # The U-Boot environment is written directly to the SDCard/eMMC. It is not
 # in any partition
-define(UBOOT_ENV_OFFSET, 1792)
+define(UBOOT_ENV_OFFSET, 8192)
```

Comment:
This was out of sync with `fwup.conf` - I haven't figured out it that was on purpose yet, but this change gets in sync